src/secrets.js

```diff
@@ -1,6 +1,7 @@
 export const PROJECT_ID = process.env.REACT_APP_PROJECT_ID;
 export const API_KEY = process.env.REACT_APP_API_KEY;
 export const APP_ID = process.env.REACT_APP_APP_ID;
+export const AUTH_DOMAIN = process.env.REACT_APP_AUTH_DOMAIN;
```

Comment:
[OOS] We should really prioritize some work to use the 'automatic' setup option and remove the need for us to configure/pass through all of this stuff from ENV vars (with an option to fall back to/override using the ENV vars still probably)

See https://firebase.google.com/docs/hosting/reserved-urls?authuser=2#sdk_auto-configuration